uditagrawal08/task4.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        section{padding-top: 50px;
+            text-align: center;
+            margin-bottom: 500px;
+            margin-top: 50px;
+        }
```

Comment:
Nice, Great hack @uditagrawal08!
I suggest you to try `height: 100vh` next time, which means 100% vertical height.
It will allow you to set height according to screen sizes.